tutorials/terraform/README.md

```diff
@@ -1,14 +1,33 @@
 # Terraform configuration files for Yandex Cloud practical guides
 
-* [clickhouse-hybrid-storage.tf](./clickhouse-hybrid-storage.tf)
+All files contains resources:
 
-    * [Using hybrid storage](https://cloud.yandex.com/en/docs/managed-clickhouse/tutorials/hybrid-storage).
-    * [Использование гибридного хранилища](https://cloud.yandex.ru/docs/managed-clickhouse/tutorials/hybrid-storage).
+* Network
+* Subnets
+* Security groups
+* Rules for security groups
 
-### Getting data from Managed Service for Apache Kafka
+## Using hybrid storage for Yandex Managed Service for ClickHouse cluster
+
+[Using hybrid storage](https://cloud.yandex.com/en/docs/managed-clickhouse/tutorials/hybrid-storage) / [Использование гибридного хранилища](https://cloud.yandex.ru/docs/managed-clickhouse/tutorials/hybrid-storage).
+
+**FIles**:
+
+* [clickhouse-hybrid-storage.tf](./clickhouse-hybrid-storage.tf) — Yandex Managed Service for ClickHouse cluster with hybrid storage.
```

Comment:
Yandex Managed Service for ClickHouse cluster with hybrid storage. -> Yandex Managed Service for ClickHouse cluster with a hybrid storage.